package.json

```diff
@@ -36,11 +39,14 @@
   },
   "homepage": "https://github.com/mapbox/assembly#readme",
   "dependencies": {
+    "@mapbox/batfish": "^0.5.0",
     "autoprefixer": "^6.6.1",
     "concat-with-sourcemaps": "^1.0.4",
     "cp-file": "^4.1.1",
     "csso": "^3.1.1",
+    "es6-promise": "^4.1.1",
```

Comment:
Oh yeah, good catch, I meant to come back and debug this... removed & working as expected!